packages/toolpad-app/src/appDom/index.ts

```diff
@@ -55,6 +55,7 @@ export interface AppDomNodeBase {
   readonly id: NodeId;
   readonly type: AppDomNodeType;
   readonly name: string;
+  readonly displayName?: string;
```

Comment:
This should be under `attributes` like `title` and `display` are.

Reply:
https://github.com/mui/mui-toolpad/discussions/2526
![image](https://github.com/mui/mui-toolpad/assets/27261522/6d9c43eb-c20d-486c-8296-8c5beb4e5d2c)

I think we could have discussed there

Reply:
I see, nevermind then! We can keep that as it is.

Reply:
That screenshot talks about the page.yml. In the appDom it probably makes more sense to put it under attributes